control/control.go

```diff
@@ -572,6 +572,12 @@ func (p *pluginControl) verifySignature(rp *core.RequestedPlugin) (bool, serror.
 }
 
 func (p *pluginControl) returnPluginDetails(rp *core.RequestedPlugin) (*pluginDetails, serror.SnapError) {
+	if rp.Uri() != nil {
+		return &pluginDetails{
+			Uri:          rp.Uri(),
+			IsAutoLoaded: true,
```

Comment:
Remove the line `IsAutoLoaded: true` after this field is removed from the struct in plugin_manager.go.